<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="assemblyText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>321</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuAsm">
    <property name="title">
     <string>Asm</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionGenerate_bin"/>
    <addaction name="actionGenerate_coe"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Disasm</string>
    </property>
    <addaction name="actionDiscompile_2"/>
    <addaction name="actionLoad_coe"/>
    <addaction name="actionLoad_bin"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAsm"/>
   <addaction name="menuFont"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
  </action>
  <action name="actionDiscompile">
   <property name="text">
    <string>Discompile</string>
   </property>
  </action>
  <action name="actionFont_Size">
   <property name="text">
    <string>Font Size</string>
   </property>
  </action>
  <action name="actionFont_Color">
   <property name="text">
    <string>Font Color</string>
   </property>
  </action>
  <action name="actionChange_Color">
   <property name="text">
    <string>Change Color</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDiscompile_2">
   <property name="text">
    <string>Discompile</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionGenerate_bin">
   <property name="text">
    <string>Generate bin</string>
   </property>
  </action>
  <action name="actionGenerate_coe">
   <property name="text">
    <string>Generate coe</string>
   </property>
  </action>
  <action name="actionLoad_coe">
   <property name="text">
    <string>Load coe</string>
   </property>
  </action>
  <action name="actionLoad_bin">
   <property name="text">
    <string>Load bin</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
